jetty-http2/http2-server/pom.xml

```diff
@@ -78,4 +82,34 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>run-spec-server</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>spec-server-run</id>
+                <phase>test</phase>
+                <configuration>
+                  <mainClass>org.eclipse.jetty.http2.server.H2SpecServer</mainClass>
+                  <classpathScope>test</classpathScope>
+                  <arguments>
+                    <argument>8080</argument>
```

Comment:
Pick a different port.
Port 8080 is often used by other processes on the same machine.
How about 28888 ?

Reply:
this one is just for manual local test and activated by test. The real test part of the build is using random port

Reply:
done